core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.ts

```diff
@@ -740,6 +784,62 @@ export class EditEmaEditorComponent implements OnInit, OnDestroy {
                     this.host
                 );
             },
+
+            [CUSTOMER_ACTIONS.COPY_CONTENTLET_INLINE_EDITING]: () => {
+                this.store.setEditorMode(EDITOR_MODE.EDIT);
+                const payload = <{ dataset: InlineEditingContentletDataset }>data.payload;
+
+                this.dotCopyContentModalService
+                    .open()
+                    .pipe(
+                        switchMap(({ shouldCopy }) => {
+                            if (!shouldCopy) {
+                                return of(payload);
+                            }
+
+                            return this.handleCopyContent();
+                        })
+                    )
+                    .subscribe((res: DotCMSContentlet) => {
+                        const updatedDataset = {
+                            inode: res.inode || payload.dataset.inode,
+                            fieldName: payload.dataset.fieldName,
+                            mode: payload.dataset.mode,
+                            language: payload.dataset.language
+                        };
+
+                        this.inlineEditingService.setTargetInlineMCEDataset(updatedDataset);
+                        if (!res.dataset) {
+                            this.store.reload({
+                                params: this.queryParams,
+                                whenReloaded: () => {
```

Comment:
Didn't we going to deprecate the `whenReloaded`?

Reply:
Yes! I just remove the callback, is not needed for this case